v1.1/ProvideChargingInformationRequest.json

```diff
@@ -422,22 +439,27 @@
 				},
 				"insideTemperature": {
 					"type": "number",
+					"description": "Temperature inside charging equipment in [°C]",
 					"additionalProperties": false
 				},
 				"outsideTemperature": {
 					"type": "number",
+					"description": "Temperature outside charging equipment in [°C]",
 					"additionalProperties": false
 				},
 				"connectorTemperature": {
 					"type": "number",
+					"description": "Temperature of connector of charging equipment in [°C]",
 					"additionalProperties": false
 				},
 				"presentPowerLimit": {
 					"type": "number",
+					"description": "Optional power limit in [kWh] currently set by charging management system.",
```

Comment:
Typo `presentPowerLimit`. Ein Leistunglimit muss in Watt oder Kilowatt angegeben werden.